emails/templates/welcome_on_signup.txt

```diff
@@ -0,0 +1,11 @@
+[[Subject .Subject "Welcome to Grafana"]]
+
+Hi [[.Name]],
+
+Welcome! Ready to start building some beautiful metric and analytic dashboards?
+
+If you are new to Grafana please read the Getting Started guid than can be found on https://grafana.com/docs/grafana/latest/getting-started/getting-started/.
```

Comment:
```suggestion
If you are new to Grafana, please read the [getting started guide](https://grafana.com/docs/grafana/latest/getting-started/getting-started/).
```

Reply:
@oddlittlebird this is a plain-text file, so Markdown formatting is not supported.

Reply:
Fixed typos but did not use Markdown syntax

Reply:
The guide is titled *Getting started with Grafana*, rather than *getting started guide*.

Please change to, "If you are new to Grafana, refer to the *Getting started with Grafana* guide.